pkgs/development/python-modules/pyimpfuzzy/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
+, pefile
+}:
+buildPythonPackage rec {
+  pname = "pyimpfuzzy";
+  version = "0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "da9796df302db4b04a197128637f84988f1882f1e08fdd69bbf9fdc6cfbaf349";
+  };
+
+  buildInputs = [
+    pkgs.ssdeep
```

Comment:
```suggestion
    ssdeep
```

Reply:
This is actually necessary, because the `ssdeep` Python module does not give us what we need:

```
impfuzzy_util.c:2:10: fatal error: fuzzy.h: No such file or directory
    2 | #include <fuzzy.h>
      |          ^~~~~~~~~
```

Reply:
Though if there's a better way to do this, I'm all ears.

I don't know if `ssdeep` as an input and then `callPackage ..../pyimfuzzy { inherit (pkgs) ssdeep; };` is any better, though.